cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/PredicateFunctionsTest.scala

```diff
@@ -179,7 +179,7 @@ class PredicateFunctionsTest extends DocumentingTest {
     }
     section("none()", "functions-none") {
       p("""The function `none()` returns `true` if the predicate does _not_ hold for any element in the given list.
-          #`null` is returned if the list is `null` or all of its elements are `null`.""".stripMargin('#'))
+          #`null` is returned if the list is `null` or if one of the comparisons evaluates to `null` and no other has evaluated to true.""".stripMargin('#'))
```

Comment:
My experiments show

```
1. RETURN none(x IN [null] WHERE x = 1) => null
2. RETURN none(x IN [null, 1] WHERE x = 1) => false
3. RETURN none(x IN [1, null] WHERE x = 1) => false
4. RETURN none(x IN [2,null] WHERE x = 1) => null
5. RETURN none(x IN [null, 2] WHERE x = 1) => null
6. RETURN none(x IN [1, null, 2] WHERE x = 1) => false
```

This also seems to match with your sentence, so just same wording suggestion as above

```suggestion
          #`null` is returned if the list is `null`, or if the predicate evaluates to `null` for at least one element and does not evaluate to `true` for any other element.""".stripMargin('#'))
```